src/System.Runtime.Caching/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Windows_NT;
```

Comment:
we need a netcoreapp2.0 configuration also for the package to be compatible with it. So for what you're trying to achieve here it would look something like: 

```xml
<PackageConfigurations>
  netstandard;
  netcoreapp2.0-Windows_NT;
</PackageConfigurations>
<BuildConfigurations>
  $(PackageConfigurations);
  netcoreapp-Windows_NT;
</BuildConfigurations>
```

This will make the package netstandard2.0 and netcoreapp2.0 compatible which we need.

Reply:
fixed

Reply:
Thanks @danmosemsft